misc/toml-parser/src/main/java/io/ballerina/toml/internal/parser/TomlParser.java

```diff
@@ -470,10 +518,12 @@ private STNode parseStringContent() {
         STToken nextToken = peek();
         if (nextToken.kind == SyntaxKind.IDENTIFIER_LITERAL) {
             return consume();
-        } else {
-            recover(nextToken, ParserRuleContext.STRING_CONTENT);
-            return parseStringContent();
         }
+        if (nextToken.kind == SyntaxKind.DOUBLE_QUOTE_TOKEN) {
+            return STNodeFactory.createEmptyNode();
+        }
+        recover(nextToken, ParserRuleContext.STRING_CONTENT);
```

Comment:
seems this method is called within both string and literal string. if it is literal string shouldn't we need to go to recovery with LITERAL_STRING_CONTENT?